azure-mgmt-trafficmanager/src/main/java/com/microsoft/azure/management/trafficmanager/TrafficManagerEndpoint.java

```diff
@@ -48,6 +52,12 @@
     long routingPriority();
 
     /**
+     * @return the geographic location codes indicating the locations to which traffic will be distributed.
+     */
+    @Beta(Beta.SinceVersion.V1_2_0)
+    List<String> geographicLocationCodes();
```

Comment:
Why `List<String>` not `Set<String>`? Is there meaning to the order and/or duplicate values?